src/CSV.jl

```diff
@@ -1,6 +1,7 @@
 __precompile__(true)
 module CSV
 
+using Base: Callable
```

Comment:
Is this related? Otherwise, looks good.

Reply:
I believe I anticipated I would need this and then forgot to remove it. I'll take it out.